lib/java/ydoc-server/src/main/java/org/enso/ydoc/server/YdocServerImpl.java

```diff
@@ -0,0 +1,29 @@
+package org.enso.ydoc.server;
+
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+import org.enso.runner.common.WrongOption;
+import org.enso.runner.common.YdocServerApi;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@org.openide.util.lookup.ServiceProvider(service = YdocServerApi.class)
+public final class YdocServerImpl extends YdocServerApi {
+  private static final Logger log = LoggerFactory.getLogger(YdocServerImpl.class);
+
+  public YdocServerImpl() {}
+
+  @Override
+  protected AutoCloseable runYdocServer(String hostname, int port) throws WrongOption, IOException {
+    try {
+      var then = System.currentTimeMillis();
+      var ydoc = Ydoc.builder().hostname(hostname).port(port).build();
+      ydoc.start();
+      var now = System.currentTimeMillis();
+      log.warn("Ydoc server at {}:{} started in {} ms", hostname, port, now - then);
```

Comment:
With three subsequent start ups I see:
```
Ydoc server at localhost:1234 started in 452 ms
Ydoc server at localhost:1234 started in 347 ms
Ydoc server at localhost:1234 started in 327 ms
```
e.g. we can assume that the slowdown is at most half a second. That sounds more acceptable than 5s in the HotSpot JVM mode. Let the investigations continue...